.github/ISSUE_TEMPLATE/performance.yml

```diff
@@ -0,0 +1,136 @@
+description: This report is for the performance-related issue
+title: "[Performance]: "
+labels: ["performance", "support_request"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please provide all the necessary information to expedite the response.
+  - type: input
+    id: ov_version
+    attributes:
+      label: OpenVINO Version 
+      description: OpenVINO version, branch, or tag in OpenVINO GitHub
+      placeholder: 2021.4.0 LTS / Master Branch / tag 2022.3.0
+    validations:
+      required: false
+  - type: input
+    id: target_platform
+    attributes:
+      label: Target Platform
+      description: The platform that the OpenVINO source is built on
+      placeholder: 11th Gen Intel Core i7-1185G7
+    validations:
+      required: false
+  - type: dropdown
+    id: os
+    attributes:
+      label: Operating System
+      description: What OS are you using?
+      options:
+        - Ubuntu 18.04 (LTS)
+        - Ubuntu 20.04 (LTS)
+        - Windows System
+        - Red Hat Enterprise Linux 8
+        - Android System
+        - Raspbian Stretch OS
+        - macOS Systems for Intel CPU
+        - macOS Systems for Apple Silicon
+        - WebAssembly
+        - Other (Please specify in description)
+    validations:
+      required: true
+  - type: dropdown
+    id: device_use
+    attributes:
+      label: Device used for inference
+      description: What hardware are you using for inference?
+      options:
+        - CPU
+        - iGPU
+        - dGPU
+        - VPU
+    validations:
+      required: false
+  - type: dropdown
+    id: openvino_installation
+    attributes:
+      label: OpenVINO installation 
+      description: How do you install OpenVINO on your system?
+      options:
+        - PyPi
+        - Docker
+        - Build from source
+        - Other
+    validations:
+      required: true
+  - type: dropdown
+    id: openvino_api
+    attributes:
+      label: Programming Language 
+      description: What is the programming language you use in your performance test? 
+      options:
+        - Python
+        - C++
+        - Other
+    validations:
+      required: true
+  - type: dropdown
+    id: architecture
+    attributes:
+      label: Hardware Architecture 
+      description: What is your hardware architecture used in this test?
+      options:
+        - Intel
+        - ARM
+        - Other
+    validations:
+      required: true
+  - type: input
+    id: model_name
+    attributes:
+      label: Model used
+      description: Please provide us the link to your model in the description
+      placeholder: ResNet50 / YOLOv4
+    validations:
+      required: true
+  - type: dropdown
+    id: model_quantized
+    attributes:
+      label: Quantized model?
+      description: Is your model quantized with model optimized?
+      options:
+        - 'Yes'
+        - 'No'
+    validations:
+      required: true
+  - type: textarea
+    id: performance_description
```

Comment:
Please add application information, benchmark_app or customer application. 

If this is issue on customer application. could this issue be reproduced on benchmark_app?

Reply:
I put it as description below: 
 Please provide a detailed description of what happened
 let us know the application you use for performance tests
 e.g., hello_classification.py / benchmark_app / own test script
 if you are using your test script, could this reproduce using benchmark_app?